gem/earlgrey.gemspec

```diff
@@ -0,0 +1,43 @@
+#
+#  Copyright 2016 Google Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+require_relative 'lib/earlgrey/version'
+
+Gem::Specification.new do |s|
+  s.required_ruby_version = '>= 2'
+
+  s.name          = 'earlgrey'
+  s.version       = EarlGrey::VERSION
+  s.license       = 'Apache-2.0'
```

Comment:
LICENSE file has both Apache 2.0 and CC-BY 4.0. I don't think we need CC-BY 4.0 in the gem/ folder unless you have a good reason for it.

Either way, this should be updated to correctly reflect the licenses used.